sections_flask-jwt-extended/README.md

```diff
@@ -13,33 +13,34 @@ This section presents the basic usage of an active flask JWT extension called `f
 ## Related Resources
 
 ### UserLogin
+
 - `POST: /login`
-    - Description: authenticate a user and ,if authenticated, respond with an access token and a refresh token.
+    - Description: authenticate a user and ,if authenticated, respond with a fresh access token and a refresh token.
```

Comment:
I changed the workflow slightly. We simply return a fresh access token when the user authenticates via credentials. We don't need a separate `/fresh-login` endpoint anymore.